src/android/UpdateHashUtils.java

```diff
@@ -33,16 +52,34 @@ public static String getBinaryHash(Activity activity) throws IOException, NoSuch
         return computeHash(new ByteArrayInputStream(manifestString.getBytes()));
```

Comment:
We should log manifest content here like this: `log("Manifest string: " + updateContentsManifestString);`